content/docs/user-guide/build/configure-and-build.md

```diff
@@ -64,7 +71,7 @@ The typical CMake command used to configure a project looks like the following:
 {{% tab name="Windows" %}}
 
 ```cmd
-cmake -B build/windows_vs2019 -S . -G "Visual Studio 16" -DLY_3RDPARTY_PATH=<downloadable-packages-directory>
+cmake -B build/windows -S . -G "Visual Studio 16" -DLY_3RDPARTY_PATH=<downloadable-packages-directory>
```

Comment:
Minor nit that we can handle later if needed - should we have three tabs here, two for Windows with different VS versions? Or should we just update to "Visual Studio 17" in stabilization / at the time of merge?

Reply:
Having 3 tabs is not a bad idea. I'd prefer that. I'll create a GHI for follow up.

Reply:
https://github.com/o3de/o3de.org/issues/2026